services/identity/src/main/java/com/crapi/controller/AuthController.java

```diff
@@ -177,4 +179,24 @@ public ResponseEntity<JwtResponse> loginWithTokenV2(
     }
     return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(jwt);
   }
+
+  /** @return success or failure of password updation. */
+  @PostMapping("/reset")
+  public ResponseEntity<?> resetPassword() {
+
+    Map<String, String> testUsers = new HashMap<String, String>();
+    testUsers.put("adam007@example.com", "adam007!123");
```

Comment:
Should we move it somewhere like the constants directory? So that we dont have to maintain this at two different places

Reply:
done